Exercise7.js

```diff
@@ -0,0 +1,10 @@
+/*
+function mult(x,y){
+  return x*y;
+}
+
+*/
```

Comment:
Would you please consider removing the comments they are serving no role in this file.